js/src/components/SchemaTab.js

```diff
@@ -0,0 +1,237 @@
+import React, { Fragment } from "react";
+import styled from "styled-components";
+import { Select } from "@yoast/components/src/select/Select";
+import { __, sprintf } from "@wordpress/i18n";
+import { createPortal } from "react-dom";
+import HelpIcon from "@yoast/components/src/help-icon/HelpIcon";
+import SidebarCollapsible from "./SidebarCollapsible";
+import PropTypes from "prop-types";
+import interpolateComponents from "interpolate-components";
+
+const SchemaContainer = styled.div`
+	padding: 16px;
+`;
+
+const schemaPageOptions = [
+	{
+		name: __( "Web Page", "wordpress-seo" ),
+		value: "option-1",
+	},
+	{
+		name: __( "Item Page", "wordpress-seo" ),
+		value: "option-2",
+	},
+	{
+		name: __( "About Page", "wordpress-seo" ),
+		value: "option-3",
+	},
+	{
+		name: __( "FAQ Page", "wordpress-seo" ),
+		value: "option-4",
+	},
+	{
+		name: __( "QA Page", "wordpress-seo" ),
+		value: "option-5",
+	},
+	{
+		name: __( "Profile Page", "wordpress-seo" ),
+		value: "option-6",
+	},
+	{
+		name: __( "Contact Page", "wordpress-seo" ),
+		value: "option-7",
+	},
+	{
+		name: __( "Medical Web Page", "wordpress-seo" ),
+		value: "option-8",
+	},
+	{
+		name: __( "Collection Page", "wordpress-seo" ),
+		value: "option-9",
+	},
+	{
+		name: __( "Checkout Page", "wordpress-seo" ),
+		value: "option-10",
+	},
+	{
+		name: __( "Real Estate Listing", "wordpress-seo" ),
+		value: "option-11",
+	},
+	{
+		name: __( "Search Results Page", "wordpress-seo" ),
+		value: "option-12",
+	},
+	{
+		/* translators: %1$s expands to "- " (a hyphen and a space), %2$s expands to " -" (a space and a hyphen) */
+		name: sprintf( __( "%1$sNone%2$s", "wordpress-seo" ), "- ", " -" ),
```

Comment:
Why would we translate the hyphen position? Are there languages which put hyphens in different places?


Another note, I seem to remember using `__` outside of the render loop would give trouble. I'm not sure if that is still the case or even why that was anymore. Worth testing I think.

Reply:
I don't know why using `__` outside the render method would be a problem. Looking at [the WordPress documentation](https://developer.wordpress.org/block-editor/developers/internationalization/) indicates that this is not a problem.

For the lazy, they give the following example:
```
import { __ } from '@wordpress/i18n';
import { registerBlockType } from '@wordpress/blocks';
 
registerBlockType( 'myguten/simple', {
    title: __( 'Simple Block', 'myguten' ),
    category: 'widgets',
 
    edit: () => {
        return (
            <p style="color:red">
                { __( 'Hello World', 'myguten' ) }
            </p>
        );
    },
 
    save: () => {
        return (
            <p style="color:red">
                { __( 'Hello World', 'myguten' ) }
            </p>
        );
    },
} );
```
Which uses `__` outside the render method.